architecture/core_objects/builds.adoc

```diff
@@ -169,24 +170,22 @@ following format:
 
 You can define multiple Docker registry entries in this file. Alternatively, you
 can also add authentication entries to this file by running the `docker login`
-command. The file will be created if it does not exist.
-
-Kubernetes provides the
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
-resource, which is used to store your configuration and passwords. You must
-first create the `*Secret*` before builds can use your *_.dockercfg_* file for
-pushing the output image:
+command. The file will be created if it does not exist. Kubernetes provides
+https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secrets],
+which are used to store your configuration and passwords.
 
+. Before builds can use your *_.dockercfg_* file for pushing the output image,
+create the `*Secret*`:
++
 ====
 ----
 $ osc create -f secret.json
 ----
 ====
 
-The `*data*` field for the `*Secret*` object must contain the `*dockercfg*` key
-with the value set to the base64-encoded content of the *_.dockercfg_* file. For
-example:
-
+. The `*data*` field in the `*Secret*` must contain the `*dockercfg*` value set
+to the base64-encoded content of the *_.dockercfg_* file. For example:
++
```

Comment:
This question might need to be reference in the SSH procedure as well, but is this missing a step? In the SSH procedure we use the `base64` command to find a value to put in the config file. Should a similar step be here?